### ISSUE TITLE
Fix missing image ALT text

### ISSUE THREAD

**Issue by fharper** [konstructio/kubefirst-docs#647]
### What type of suggestions?

Something's not working

### What page is concerned?

ALL

### What version of the docs?

ALL

### What is the issue?

We cannot upgrade the `DavidAnson/markdownlint-cli2-action` workflow we use to check Markdown validity as it now detect properly missing ALT text, and we are missing a lot. So it's a great news since our docs will be more accessible, but we need to fix those before upgrading so all the new PR don't fail with this action.

Here just part of the errors:
![CleanShot 2024-02-28 at 14 21 48@2x](https://github.com/kubefirst/docs/assets/1170989/8a8a1a56-c39d-49f5-aa41-215499b752e3)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct
